Jenkinsfile

```diff
@@ -17,5 +17,21 @@ pipeline {
                 }
             }
         }
+        stage('Deliver Docker image') {
```

Comment:
I didn't change all builds yet but please check the java version you need for compiling and running tests with maven above.

Then you can change 

```
agent any
```
to 
```
agent {
  label 'jdk17'
}
```
or jdk11 depending your needs. You can leave any if your build is able to work on both though

Reply:
The failed build for example ran on a jdk11 worker

Reply:
This should work on the jdk11 workers. Seems the worker can not download the LSC dependency, will look tmr.

Reply:
Agree the build uses lower JDK versions but we can choose a higher JRE without troubbles.